src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/RegionDropdown.tsx

```diff
@@ -0,0 +1,67 @@
+import React, {FC, useCallback, useContext, useMemo} from 'react'
+import {ComponentStatus, Dropdown} from '@influxdata/clockface'
+
+// Components
+import {Cluster} from 'src/client/unityRoutes'
+import {CreateOrgContext} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/CreateOrganizationContext'
+
+const SINGLE_PROVIDER_WIDTH = 375
+
+export const RegionDropdown: FC = () => {
```

Comment:
I was curious about the need for this dropdown as it's own component?

Reply:
i like to keep the parent container with less code and only the components or gluing code. this been said, i am happy to move it up but i feel that'd clutter the parent a little more as it's already big enough.

Reply:
let me know your thoughts

Reply:
I don't know what convention we want to follow, I definitely hear your point that it will bloat up the parent file. I'll let @hoorayimhelping make a call. 

The only reason I bring this up was in the past we had a few split components that when we needed to add any new functionality, a lot of prop drilling was needed.

Reply:
as discussed in code review, i am going to handle this in ffaf